rules/Arguments/ArgumentDefaultValueReplacer.php

```diff
@@ -166,7 +159,7 @@ private function processArrayReplacement(
      */
     private function resolveArgumentValuesToBeforeRecipe(
         array $argumentNodes,
-        ReplaceArgumentDefaultValueInterface $replaceArgumentDefaultValue
+        \ReplaceArgumentDefaultValueInterface $replaceArgumentDefaultValue
```

Comment:
debuggin: 

```php
$type = $this->staticTypeMapper->mapPhpParserNodePHPStanType($phpParserTypeNode);
 dump($type);
 ```

 on `ClassMethodParamTypeCompleter` seems got type:

```php
Rector\StaticTypeMapper\ValueObject\Type\FullyQualifiedObjectType #78514
   subtractedType: null
   cachedParent: null
   cachedInterfaces: null
   currentAncestors: array (0)
   className: 'ReplaceArgumentDefaultValueInterface'
   classReflection: null
```

which namespaced in use not included.

https://github.com/rectorphp/rector-src/blob/1836de97836c78ae4c85c1fdce3664688fb1011f/rules/TypeDeclaration/NodeAnalyzer/ClassMethodParamTypeCompleter.php#L63

Reply:
tweak by check virtual node https://github.com/rectorphp/rector-src/pull/2701/commits/39e3969a2a4884cf6bceeee558f36ceef4f4c5d0